aws-ts-webserver/index.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+// Get the id for the latest Amazon Linux AMI
+const ami = aws.ec2.getAmi({
+    filters: [
+        { name: "name", values: ["amzn-ami-hvm-*-x86_64-ebs"] },
+    ],
+    owners: ["137112412989"], // Amazon
+    mostRecent: true,
+}, { async: true }).then(result => result.id);
```

Comment:
```suggestion
}).then(result => result.id);
```

We no longer need `async: true` post-2.0